### ISSUE TITLE
Launching PowerShell generates errors on load.

### ISSUE THREAD

**Issue by mhenderson-mesa** [PowerShell/PSReadLine#2975]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19043
BufferWidth: 120
BufferHeight: 30

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

### Screenshot

![image](https://user-images.githubusercontent.com/70279301/140927481-2a425fd8-c81e-47ab-9f41-80140c19f481.png)

### Environment data

```console
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19043
BufferWidth: 120
BufferHeight: 30
```

### Steps to reproduce

Launch PowerShell 7.2 in Window Terminal or VS Code terminal.

### Expected behavior

Expecting to launch the application.

### Actual behavior

Unable to use. I know that's pretty vague, after upgrading this morning I was simply unable to use version 7.2

Resolved issue referring to Scott Hanselman's [Blog Entry](https://www.hanselman.com/blog/powershell-720-could-not-load-type-systemmanagementautomationsubsystempredictionresult)

**Comment by CptChaos** [comment]
See https://github.com/PowerShell/PSReadLine/issues/2974, this is the same issue. You need to update PSReadLine, I posted a command to simply update PSReadLine to the latest version there.

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
